src/getting_started/platform-specific_setup.md

```diff
@@ -10,6 +10,12 @@ Depending on what OS you are running, you might require an extra step or two.
   Vulkan implementation. There are a few attempts at pure-rust alternatives to
   this tool in the works and we hope to switch to one of these in the near
   future to avoid the need for these extra dependencies.
+  Rustup need the  the following packages:
+  * `bash`
+  * `build-essentials`
+  * `python`
+  * `cmake`
+  * `wget` or `curl`
```

Comment:
If you only use one line break, the markdown renderer will not create a new paragraph. Right now if you check the rendered version it looks like these rustup deps are part of the Vulkan paragraph.

`wget` and `build-essentials` are not needed by *all* platforms, and I don't think that `wget` is needed at all, whereas `curl` is required. Also, rustup does not need `cmake` or `python`, only `vulkano` requires these due to the `shaderc` dependency.

Reply:
These are all the dependencies either required by me building nannou or as reported by others.